package.json

```diff
@@ -31,11 +41,16 @@
     }
   },
   "devDependencies": {
-    "tree-sitter-cli": "^0.20.7",
-    "prebuildify": "^6.0.0"
+    "prebuildify": "^6.0.0",
+    "tree-sitter-cli": "^0.22.2"
```

Comment:
Updated it based on this request from a previous PR https://github.com/RubixDev/tree-sitter-asm/pull/10#issuecomment-2010976866